web/pgadmin/tools/sqleditor/static/js/components/QueryToolDataGrid/index.jsx

```diff
@@ -344,11 +427,16 @@ function formatColumns(columns, dataChangeStore, selectedColumns, onSelectedColu
 
   for(const [idx, col] of retColumns.entries()) {
     col.renderHeaderCell = HeaderRenderer;
+    col.renderCell = (props)=>{
```

Comment:
do we really need this? react-data-grid has `onCellKeydown` prop. We should be able to use that I think.

Reply:
I already tried that, but I wasn't successful.
Encountered issue on selecting all rows with react-data-grid hook `onRowSelectionChange`.